htdocs/web_portal/views/start_page.php

```diff
@@ -34,14 +35,10 @@
 
     <!-- map Block -->
     <?php if($params['showMap']): ?>
-        <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?sensor=false&key=<?php echo $params['googleAPIKey'];?>">
-        </script>
-        <!--This script provides the marker clustering functionality comment out the cluster line in googleSiteMap.js and this script to disable it-->
-        <!--<script type="text/javascript" src="<?php echo \GocContextPath::getPath()?>javascript/googleMapClusterer.js">
-        </script>-->
-        <script type="text/javascript" src="<?php echo \GocContextPath::getPath()?>javascript/googleSiteMap.js">
-        </script>
-        <div id="GoogleMap" style="width:100%;height:400px;"></div>
-        
+      <!-- Use a web based leaflet.js to avoid distributing leaflet.js. -->
```

Comment:
Are you sure we want to do this? What are the disadvantages of having it as part of our codebase? I worry that by pulling code from a third party we provide a method for someone to inject code into GOCDB. But maybe there are good reasons an this is a very small risk in the scheme of things? @jrha?

Reply:
See also #140